src/Features/VisualBasic/Portable/ConvertForEachToFor/VisualBasicConvertForEachToForCodeRefactoringProvider.vb

```diff
@@ -165,5 +171,29 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.ConvertForEachToFor
                 type = Nothing
             End If
         End Sub
+
+        Protected Overrides Function CheckIfNextStatementIsSupported(foreachVariable As ILocalSymbol, foreachOperation As IForEachLoopOperation, foreachStatement As ForEachBlockSyntax) As Boolean
```

Comment:
Probably just rename the method to `IsSupported` or `IsValid` or some such, we don't need `NextStatement` in the name as that is just VB specific implementation details for this method.